administrator/components/com_templates/Model/TemplateModel.php

```diff
@@ -76,10 +82,10 @@ public function getFiles()
 		if ($template = $this->getTemplate())
 		{
 			jimport('joomla.filesystem.folder');
-			$app    = \JFactory::getApplication();
```

Comment:
Namespacing should be done against the main repo. Like that we can keep the pr's here small and will have less possible merge conflicts.

Reply:
should I revert this?

Reply:
If ok for you, then I would revert all the namespacing. When we create the pr then for the main repo it is easier to review then.